ide-common/src/main/kotlin/org/digma/intellij/plugin/ui/service/ErrorsViewService.kt

```diff
@@ -163,7 +164,7 @@ class ErrorsViewService(project: Project) : AbstractViewService(project) {
 
 
     private fun computeErrorsPreviewCount(documentInfoContainer: DocumentInfoContainer): Int {
-        return documentInfoContainer.allInsights.stream().filter{ it is ErrorInsight }.count().toInt()
+        return documentInfoContainer.countInsightsByType(InsightType.Errors)
```

Comment:
Shouldn't this method access `ErrorInsight.errorCount`?

Reply:
nope, the `.count()` is on java stream after the filter been applied.